docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -732,11 +733,42 @@ The fields are:
 - `cursor_path` (str): The JSONPath to the field within each item in the list. This is the field that will be used to track the incremental loading. In the example above, it's `"created_at"`.
 - `initial_value` (str): The initial value for the cursor. This is the value that will initialize the state of incremental loading.
 - `end_value` (str): The end value for the cursor to stop the incremental loading. This is optional and can be omitted if you only need to track the start condition. If you set this field, `initial_value` needs to be set as well.
+- `convert` (callable): A callable that converts the cursor value into the format that the query parameter requires. For example, a UNIX timestamp can be converted into an ISO 8601 date or a date can be converted into `created_at+gt+{date}`.
 
 See the [incremental loading](../../general-usage/incremental-loading.md#incremental-loading-with-a-cursor-field) guide for more details.
 
 If you encounter issues with incremental loading, see the [troubleshooting section](../../general-usage/incremental-loading.md#troubleshooting) in the incremental loading guide.
 
+### Convert the incremental value before calling the API
```

Comment:
@burnash shall we discuss it here as a h3 on the same level as the incremental param and the incremental resource config? Advantage: DRY. We only explain it once.

Or would you prefer to cut and paste the examples into the other two h3 which deal with the two setup possibilities in general?

Reply:
I think it's good to have it on the same level. No need to repeat that twice. Especially since it's an advanced topic  which requires elaboration and examples.